docs/reference/commandline/diff.md

```diff
@@ -18,31 +18,48 @@ keywords: "list, changed, files, container"
 ```markdown
 Usage:  docker diff CONTAINER
 
-Inspect changes on a container's filesystem
+Inspect changes to files or directories on a container's filesystem
 
 Options:
       --help   Print usage
 ```
 
-List the changed files and directories in a container᾿s filesystem.
- There are 3 events that are listed in the `diff`:
+List the changed files and directories in a container᾿s filesystem since the
+container was created. Three different types of change are tracked:
 
-1. `A` - Add
-2. `D` - Delete
-3. `C` - Change
+| Symbol | Description                     |
+|--------|---------------------------------|
+| `A`    | A file or directory was added   |
+| `D`    | A file or directory was deleted |
+| `C`    | A file or directory was changed |
 
-For example:
+You can use the full or shortened container ID or the container name set using
+**docker run --name** option.
 
-    $ docker diff 7bb0e258aefe
+# EXAMPLES
```

Comment:
oh, this needs to be in sentence-case, and a H2 for the online docs (all-caps is just for formatting in the man pages)